orchestration/templates/ingest-clinvar-archive.yaml

```diff
@@ -162,8 +148,23 @@ spec:
                   value: '{{ .Values.gcs.bucketName }}'
                 - name: gcs-file-path
                   value: '{{ "{{inputs.parameters.gcs-prefix}}" }}/raw/{{ $localXmlName }}'
-          {{- $extractionPvc := "{{tasks.generate-extraction-volume.outputs.parameters.pvc-name}}" }}
```

Comment:
I don't see this replaced anywhere, was it just a duplicate definition?

Reply:
yes